Formula/h/harpwise.rb

```diff
@@ -0,0 +1,33 @@
+require "fileutils"
+
+class Harpwise < Formula
+  desc "Harmonica practice-utility for the command-line using microphone and speaker"
+  homepage "https://github.com/marcIhm/harpwise"
+  url "https://github.com/marcihm/harpwise/archive/4.4.4.tar.gz"
+  sha256 "7caae0c4732bb188d164d86a4cceb2aa7b4f09859b945bc86f42d31500571e83"
+  license "MIT"
+
+  # dependencies ordered according to "brew style harpwise"
+  depends_on "aubio"
+  depends_on "coreutils"
+  depends_on "figlet"
+  depends_on "ruby"
+  depends_on "sox"
+  depends_on "toilet"
+
+  def install
+    # harpwise is written in ruby, but is not a gem.
+    man1.install "man/harpwise.1" => "harpwise.1"
+    mkdir "bin"
```

Comment:
```suggestion
```

Reply:
This mkdir happens within the downloaded and unpacked archive; it is needed for the following File.write to work.

Reply:
'mkdir "bin"' is gone, as I now create the stub within libexec.